### ISSUE TITLE
why is n24 capped at 3 on cps.csv?

### ISSUE THREAD

**Issue by MattHJensen** [open-source-economics/taxdata#131]
Why are there no values of n24 over 3 in the cps.csv file? SOI censors the PUF at 3, but why is the CPS-based file capped? 

cc @andersonfrailey @martinholmer @evtedeschi3

**Comment by evtedeschi3** [comment]
Looks like it's [explicitly capped](https://github.com/open-source-economics/taxdata/blob/master/cps_data/finalprep.py) in the `finalprep.py` file to match what would be seen in the `puf.csv`. See lines 144-153:

```
    # Count number of children eligible for child tax credit
    # Max of three to mach PUF version of n24
    age1 = np.where((data.ICPS03 > 0) & (data.ICPS03 <= 17), 1, 0)
    age2 = np.where((data.ICPS04 > 0) & (data.ICPS04 <= 17), 1, 0)
    age3 = np.where((data.ICPS05 > 0) & (data.ICPS05 <= 17), 1, 0)
    age4 = np.where((data.ICPS06 > 0) & (data.ICPS06 <= 17), 1, 0)
    age5 = np.where((data.ICPS07) > 0 & (data.ICPS07 <= 17), 1, 0)
    n24 = age1 + age2 + age3 + age4 + age5
    n24 = np.where(n24 > 3, 3, n24)
    data['n24'] = n24
```

This is interesting because here is an area where the CPS has more reliable data than the PUF. Why is the CPS being censored here? Since the PUF is matched with the CPS, couldn't/shouldn't the CPS be used to fill in detail for PUF tax units likely to have more than 3 adults?

**Comment by andersonfrailey** [comment]
It was capped to match the PUF file. I can update the final prep scripts to change this when I get back in the office.

**Comment by MaxGhenis** [comment]
Just checking in on this, as I'm working on a CTC analysis.

**Comment by andersonfrailey** [comment]
Thanks for bringing this back up @MaxGhenis. I'm working on an update to the CPS file that should be ready in the next day or two. I'll address this in that update as well.

**Comment by andersonfrailey** [comment]
This issue has been resolved in PR #161. Closing.
